dkobo/hub/context_processors.py

```diff
@@ -19,3 +19,14 @@ def welcome_message(request):
         return ctx
     else:
         return {}
+
+def external_service_tokens(request):
+    context = {}
+    if settings.TRACKJS_TOKEN:
```

Comment:
I'm going to merge. Since `os.environ.get()` returns `None` if the environment variable is unset , this should never be an issue. See https://github.com/kobotoolbox/dkobo/blob/tokens-context-processor/dkobo/settings.py#L25